src/A2A/A2A.csproj

```diff
@@ -10,6 +10,8 @@
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <IsPackable>true</IsPackable>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <!-- Don't suggest using ArgumentNullException.ThrowIfNull as we target NS2.0 where it's not present -->
```

Comment:
We could also enable using C# 14 and polyfill this as we did in runtime.

Reply:
@stephentoub We could, but this would require us to force the CI to use .NET 10 SDK for building and testing net8.0 and net9.0 and could increase the entry bar for new contributors. Do you want me to apply such changes?